bin/shutter

```diff
@@ -260,6 +260,7 @@ my $so   = Shutter::App::Options->new($sc, $shf);
 #set home
 #--------------------------------------
 $ENV{'HOME'} = Shutter::App::Directories::get_home_dir();
+$ENV{'CONFIG'} = Shutter::App::Directories::get_config_dir();
```

Comment:
Why does this need to be in the process environment?

Reply:
it's not needed. I'll drop it.